Formula/sratoolkit.rb

```diff
@@ -142,11 +142,11 @@ def install
          endif()
      else()
 @@ -581,7 +581,7 @@ if ( SINGLE_CONFIG )
-                 ${CMAKE_INSTALL_PREFIX}/bin/ncbi    \
-                 /etc/ncbi                           \
-                 ${CMAKE_INSTALL_PREFIX}/bin         \
--                ${CMAKE_INSTALL_PREFIX}/lib64       \
-+                ${CMAKE_INSTALL_LIBDIR}             \
-                 ${CMAKE_SOURCE_DIR}/shared/kfgsums  \
+                 ${CMAKE_INSTALL_PREFIX}/bin/ncbi \
+                 /etc/ncbi \
+                 ${CMAKE_INSTALL_PREFIX}/bin \
+-                ${CMAKE_INSTALL_PREFIX}/lib64 \
++                ${CMAKE_INSTALL_LIBDIR} \
+                 ${CMAKE_SOURCE_DIR}/shared/kfgsums \
```

Comment:
This is a false positive and will also break this patch

Reply:
if it is not `long build`, we can also do `ci-no-bottles` for the PR run to see if the patch would be broken.

Reply:
@chenrui333 I'd rather just remove the commit for now.